.deploy/nais.yaml

```diff
@@ -10,11 +10,11 @@ spec:
   ingresses:
     - {{ingress}}
   liveness:
-    path: /internal/isAlive
+    path: /internal/status/isAlive
     initialDelay: 30
     failureThreshold: 10
   readiness:
-    path: /internal/isReady
+    path: /internal/status/isAlive
```

Comment:
Generellt så tror jeg vi burde se på om det er riktig å bruke isalive for readiness, tror ikke det men vet att vi har det på alle våre apper

Reply:
Ja, det kommer litt an på hva som ønskes. Kunne det for eksempel vært aktuelt å gjøre en sjekk om det er tilgang til database i isReady i appene våre?